lib/features/pokemon_details/widgets/pokemon_physical_stats_view.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:pokedex_async_redux/api/model/model.dart';
+import 'package:pokedex_async_redux/features/pokemon_details/widgets/physical_stats_item.dart';
+import 'package:pokedex_async_redux/utils/colors.dart';
+import 'package:pokedex_async_redux/utils/constants.dart';
+import 'package:flutter/material.dart';
+
+class PokemonPhysicalStatsView extends StatelessWidget {
+  const PokemonPhysicalStatsView({
+    required this.physicalStats,
+    Key? key,
+  }) : super(key: key);
+
+  final PokemonDetails physicalStats;
```

Comment:
kindly pass only the needed properties instead of passing the whole `PokemonDetails` object, else this is gonna be overkill in a way 😉